wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests.wiki

```diff
@@ -33,9 +34,12 @@ The [[heuristic to find places][>FindingByHeuristics]] tries a number of transla
 |'''Command'''                                                             |'''Description'''                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 |open <url>                                                                |opens the specified URL                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
 |click <place>                                                             |performs mouse click at the designated place (e.g. link/button/checkbox/radio button)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+|click <place> xoffset <xoffset> yoffset <yoffset>                         |performs mouse click with offset from center at the designated place (e.g. link/button/checkbox/radio button)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
should there not be an 'at' in this command's documentation?
What would you think of renaming to `click <place> at offset X <horizontalOffset> Y <verticalalOffset>`?

Reply:
Changes done.